src/components/CreateNote/CreateNoteContainer.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  UseQueryResult,
+  useMutation,
+  useQuery,
+  useQueryClient,
+} from '@tanstack/react-query';
+import { v4 as uuid } from 'uuid';
+
+import { fetchNotesOfUser, notesActions } from 'api';
+import { INote } from 'types';
+import { useFieldValue, useToggle } from 'hooks';
+
+import CreateNote from './CreateNote';
+import { setNewId } from 'utils';
+import { useAppSelector } from 'store/hooks';
+import { getUser } from 'store/selectors';
+
+const CreateNoteContainer = () => {
+  const queryClient = useQueryClient();
```

Comment:
please create base.js file inside api folder anf move it there like folliwing:
`export const queryClient = new QueryClient();`

Reply:
and then reuse it where it's needed